### DISCUSSION TITLE
TP3 - Ejercicio 1.3 Fusionadora de arreglos

### DISCUSSION THREAD

**Discussion by AnGys09** [INGCOM-UNRN-P1/cursada-2023#113] [Preguntas y respuestas] [answered]
#### 1.3 Fusionadora de arreglos
Desarrollar una función que permita unir dos arreglos en un tercero que tenga la capacidad suficiente.

Hace falta usar para esta función la primera (`cargar_arreglo`) para pedirle los datos por teclado, o podemos usar dos arreglos cualesquiera a nuestra elección como 👍 
```C
    int arreglo_final [10];
    int primer_arreglo[5] = {0, 2, 4, 6, 8};
    int segundo_arreglo[5]  = {1, 3, 5, 7, 9};
```

**Comment by martinvilu** [comment, marked as answer]
Ambas opciones son correctas.

Mientras que la definición y carga de los arreglos esté en el `main`.
